sdk/src/secp256k1.rs

```diff
@@ -0,0 +1,134 @@
+use crate::clock::{Epoch, GENESIS_EPOCH};
+use crate::genesis_config::ClusterType;
+
+pub fn is_enabled_epoch(cluster_type: ClusterType) -> Epoch {
+    match cluster_type {
+        ClusterType::Development => GENESIS_EPOCH,
+        ClusterType::Testnet => 400, // todo: change this
```

Comment:
I'd just use `u64::MAX` for all three here.  We can enable them individually in a later PR